src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -654,8 +654,6 @@ public override void _Notification(int what)
             gui.UpdatePatchDetails(CurrentPatch);
             gui.UpdateMicrobePartSelections();
             gui.UpdateMutationPointsBar();
-
-            // TODO: AutoEvo run results summary
```

Comment:
Shouldn't this be the one fixed in #2790?

Reply:
true, moved it.